docs/HowToFetchToken.md

```diff
@@ -0,0 +1,94 @@
+# How to fetch IAS Tokens
+  ### With X.509 Client Certificate
+  <details>
+    <summary>Using curl command</summary>
+    
+❗Replace the `<<>>` placeholders with the values from the service configuration.   
+```shell script
+curl --cert certificate.pem --key key.pem -XPOST <<credentials.url>>/oauth2/token \
+  -d 'grant_type=password&client_id=<<credentials.clientid>>&username=<<your ias user>>@global.corp.sap&password=<<your ias password>>'
+```
+    
+  </details>
+  <details>
+    <summary>Using Postman command</summary>
+    
+1. Store the certificate and key into separate files in [PEM](https://datatracker.ietf.org/doc/html/rfc7468) format.
```

Comment:
This is relevant for Postman and curl for ias and xsuaa => I would add a separate section and explain in a little bit more detail...